src/design-system/styles/template/index.md.njk

```diff
@@ -0,0 +1,136 @@
+---
+title: Template
+description: GOV.UK Images
+section: Styles
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+To keep your pages consistent with the rest of GOV.UK you should use this template.
+
+Template combines the boilerplate markup and [components](../../components) needed for a basic GOV.UK page, it includes:
+
+- JavaScript that adds a `.js-enabled` class, which is required by components with JavaScript behaviour.
+- [Skip link](../../components/skip-link) component
+- [Header](../../components/header) component
+- [Footer](../../components/footer) component
+- favicon, and other related theme icons
+
+In the examples provided, we show both HTML and Nunjucks.
+
+If you are not using Nunjucks, you can use the HTML examples, you will need to update manually when new versions are released.
```

Comment:
To avoid starting with a negative, suggest this is reworded to something like 'You can use the HTML examples if you aren't able to use Nunjucks. You will need to manually [...]'